packages/modals/demo/stories/DrawerModalStory.tsx

```diff
@@ -15,7 +15,7 @@ import { IFooterItem } from './types';
 
 interface IArgs extends IDrawerModalProps {
   onClick: MouseEventHandler<HTMLElement>;
-  onClose: EventHandler<any>;
+  onClose: EventHandler<KeyboardEvent | MouseEvent>;
```

Comment:
If the prop is properly declared on `IDrawerModalProps` then can this `onClose` be removed?

Reply:
Yep! Type is covered in `IDrawerModalProps`. I'll remove it here.